src/makeRobot.js

```diff
@@ -38,6 +38,57 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
якщо назва звінної дорівнює ключу можна використовувати скорочений запис
```suggestion
    name,
    wheels,
    version,
```